src/libraries/System.Net.Quic/src/System/Net/Quic/Implementations/MsQuic/MsQuicListener.cs

```diff
@@ -116,7 +118,7 @@ private void Dispose(bool disposing)
             _disposed = true;
         }
 
-        internal override unsafe void Start()
+        private unsafe void Start()
         {
             ThrowIfDisposed();
```

Comment:
Shouldn't need to check for disposed anymore right? Are there any other refactorings that can be done here (like not storing `_listenEndPoint` etc.) now that we've coupled ctor with start?